citus.spec

```diff
@@ -8,11 +8,13 @@ Summary:	PostgreSQL-based distributed RDBMS
 Name:		%{sname}%{?pkginfix}_%{pgmajorversion}
 Provides:	%{sname}_%{pgmajorversion}
 Conflicts:	%{sname}_%{pgmajorversion}
-Version:	11.2.1.citus
+
+Version:	11.1.6.citus
 Release:	1%{dist}
 License:	AGPLv3
 Group:		Applications/Databases
-Source0:	https://github.com/citusdata/citus/archive/v11.2.1.tar.gz
+Source0:	https://github.com/citusdata/citus/archive/v11.1.6.tar.gz
+
```

Comment:
```suggestion
Source0:	https://github.com/citusdata/citus/archive/v11.1.6.tar.gz
```

Reply:
I guess there is something wrong with the Github interface. Can you remove the new empty line from here @gurkanindibay ?